### PR TITLE
 Add instruction how to configure the priority of sources on the Configuring Source Priority Algorithm page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request adds instruction on how to configure the priority of sources on the Configuring Source Priority Algorithm page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/catalog/inventory-configure-source-priority.html

### CODE REVIEW

File: src/catalog/inventory-configure-source-priority.md [hunk 1]
```diff
@@ -16,8 +16,23 @@ When run, the algorithm:
...
 
+## Configure the priority of sources
+
+1. On the _Admin_ sidebar, go to **Stores** > **Inventory** > **Stocks**.
+
+1. Open a stock in **Edit** mode and scroll down the page to the _Sources_ area.
+
+1. Click **Assign Sources**.
+
+    This displays the **Assign Sources page**. Select the checkbox for the required source and click <span class="btn">Done</span> to assign a source to the stock.
```
Comment (danidelcar) [inline src/catalog/inventory-configure-source-priority.md:27]: This displays the **Assign Sources** view. Select the checkbox for the required source and click **Done** to assign a source to the stock.

**Review by danidelcar — APPROVED** [review]
approved, please mind a minor change I mentioned. Thanks!

File: src/catalog/inventory-configure-source-priority.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/catalog/inventory-configure-source-priority.md]: Thanks, Done ✅ 
Please, approve.

**Review by danidelcar — APPROVED** [review]

**Comment by danidelcar** [comment]
great @ZhovnirMaria , thanks!

**Comment by danidelcar** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@danidelcar the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1212`.

**Pull request merged**